### PR TITLE
Intro

### PR DESCRIPTION
**Pull request by nemchik**
Tagging @angrycuban13 

Lots of changes in here. Mostly formatting. The intro page now has content filled and I'm looking for feedback on that. Demo: https://mkdocs.nemplex.com/introduction/

### CODE REVIEW

**Comment by angrycuban13** [comment]
I think you did a great job with the intro page. 

I almost feel like the intro page could replace the "Home" page since there is a lot of the same content on there. Just add the pretty graphics to "Intro" and it is very ELI5.

**Comment by nemchik** [comment]
I'm trying to keep the Home page a copy of the readme in the main repo.

https://github.com/GhostWriters/DockSTARTer.com/blob/master/docs/index.md
https://github.com/GhostWriters/DockSTARTer/blob/master/README.md

This way I can just copy/paste both and they stay the same. We could instead force people to visit the .com page to see the instructions to be able to install, but I like that the install instructions are small and clean and were meant to be displayed front and center when you come across the project.

So I'm kind of torn between expanding the home/index/readme to be longer and more wordy (merging the intro page), or having similar information in two places (keeping the intro page separate, maybe even expanding it more). I'm leaning towards the later.

**Comment by angrycuban13** [comment]
Yeah I think having a loooong "README" might cause people to skip reading certain things. I agree with you keep the "Home" page a copy of the readme in the repo. Whoever visits https://dockstarter.com can be automatically directed to the intro page rather than "Home".

**Comment by nemchik** [comment]
I added a commit https://github.com/GhostWriters/DockSTARTer.com/pull/26/commits/c2d9a13cbb873425f9a8998e6c584350c91ecfee so that the intro page will be the default link at the bottom of the install instructions in the readme/index, but dockstarter.com will still use the readme/index as home.
